Lib/inspect.py

```diff
@@ -2567,30 +2567,28 @@ class _empty:
 
 
 class _ParameterKind(enum.IntEnum):
```

Comment:
Is that inheritting correct where the values are changed to str?

Reply:
Good question: only because the `__new__` uses integers as the values, and the "value" on the assignment line is saved in the `description` attribute.  Without the custom `__new__` the enum creation would have failed, since strings are not integers.